### PR TITLE
Train 114

### PR DESCRIPTION
**Pull request by lingmann**
#1080 
#1010 
#1079 
#1078 
#1075

### CODE REVIEW

**Comment by lingmann** [comment]
cc: @spahl and @adam-mesos

File: test_util/run-all [hunk 1]
```diff
@@ -68,6 +68,7 @@ exhibitor_zk_hosts: 192.168.65.50:2181
 exhibitor_zk_path: /zk-shared
 master_discovery: static
 check_time: 'false'
+enable_lb: 'false'
```
Comment (adam-mesos) [inline test_util/run-all]: @sargun Shouldn't we test both true and false? It defaults to true, but we're only testing false here?

**Comment by mellenburg** [comment]
@adam-mesos if default is true for enable_lb, then the AWS, onprem (CLI and API), and Azure tests are covering it. if anything, one of those should also be set to false

File: test_util/run-all (hunk 1 above)
Comment (mellenburg) [inline test_util/run-all]: This file only covers vagrant test, so it is being tested true else where

**Comment by jgehrcke** [comment]
I have looked at the 5 pull requests linked here and they LGTM!

Nit: I'd love to see a comment in the code that contains or links to background information on `LimitNOFILE` in Spartan's unit file.

**Pull request merged**